packager/deno.json

```diff
@@ -0,0 +1,8 @@
+{
+  "tasks": {
+    "dev": "deno run --watch main.ts"
+  },
+  "imports": {
+    "yargs": "https://deno.land/x/yargs@v17.7.2-deno/deno.ts"
```

Comment:
I could simply import most dependencies from `npm`, but Deno usually has a nicer built-in alternative. So I figured I'd properly do the migration.

Yargs is a pretty good dependency though. The only alternative worth considering that I could find is https://cliffy.io/ .